docs: manual testing report POST /user/refresh (#891)
anitab-org/mentorship-backend: docs/manual_tests/user_refresh_token.md

## Manual testing for POST /user/refresh API

> #### Dummy data used :
> **username** : testrefresh, 
> **password** : testrefresh@1234

Reviewer: ```suggestion
#### Dummy data used:
* **username**: testrefresh, 
* **password**: testrefresh@1234
```

> * _Refresh token in Authorization field is the refresh token returned on login response_

Reviewer: ```suggestion
* _Refresh token using the refresh token returned on login response_
```
I changed like this, because the sentence seemed a bit confusing when reading
